Packs/MicrosoftExchangeOnline/Integrations/SecurityAndComplianceV2/SecurityAndComplianceV2.yml

```diff
@@ -1149,9 +1146,42 @@ Msg: Export to .msg message files."
       predefined:
       - 'true'
       - 'false'
+  - arguments:
+    - description: Identity of the policy to update.
+      name: Identity
+      required: true
+    - description: Exchange locations to add to the policy.
+      isArray: true
+      name: add_exchange_locations
+    - description: Sharepoint locations to add to the policy.
+      isArray: true
+      name: add_sharepoint_locations
+    - description: Public locations to add to the policy.
+      isArray: true
+      name: add_public_locations
+    - description: Exchange locations to remove from the policy.
+      isArray: true
+      name: remove_exchange_locations
+    - description: Sharepoint locations to remove from the policy.
+      isArray: true
+      name: remove_sharepoint_locations
+    - description: Public locations to remove from the policy.
+      isArray: true
+      name: remove_public_locations
+    - description: Add a comment to existing policy.
+      name: comment
+    - auto: PREDEFINED
+      defaultValue: 'true'
+      description: Enable or disable the policy.
+      name: enabled
+      predefined:
+      - 'true'
+      - 'false'
+    description: Update inputs for case hold policies.
+    name: o365-sc-case-hold-policy-set
   dockerimage: demisto/pwsh-exchangev3:1.0.0.88371
-  runonce: true
```

Comment:
This was by design mainly for a fun little MS-EXO bug. The exo client isn't destroyed after each command run, so within the container, the client continues to grow until it consumes enough memory to crash the container. We asked if MS was going to fix it and was essentially told "Ha<sup>ha<sup>ha</sup> </sup> No. Just kill the container"